docs/settings.md

```diff
@@ -633,3 +635,142 @@ Cookie: sessionid=xxxx
 
 To use this endpoint, an application needs a valid token, but no explicit
 permission is required.
+
+## Feature flags
+
+A feature flag is a name and an associated value (boolean, number, string or a
+JSON) that can be interpreted by the apps. It can be used for giving access to
```

Comment:
We already have a feature flag of type `list` of `string` defined in cozy.yml :
```
- home_hidden_app:
  - app1
  - app2
```
Will they be converted to a json object containing a list of string ? Is it already the case ?

Reply:
JSON values can be objects or arrays. Here, this flag will be an array of strings.